lib/label_printer/label_printer/label.rb

```diff
@@ -24,7 +21,7 @@ def create_labels
       end
```

Comment:
I know this wasn't anything you touched, but the big fan of Enumerable in me:

```ruby
def create_labels
  assets.flat_map do |asset|
    Array.new(count, label(asset))
  end
end
```

Its just personal preference though. (Its probably also a bit faster, but not to a sufficient enough degree that it would be worth considering on that basis)

Actually interested to know if you think it's an improvement or not.